.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+jobs:
+  build:
+    docker:
+    - image: circleci/node:8.10
+    working_directory: ~/singnet/platform-contracts
+    steps:
+    - checkout
+    - run:
+        name: Build platform-contracts
+        command: |
+          npm install ganache-cli
+          npm install
+          npm run-script compile
```

Comment:
1. ```npm run test``` actually does ```rm -rf build``` before running ```truffle test```: https://github.com/singnet/platform-contracts/blob/4ef66e32020af51133ca80d79d39b1e025328898/package.json#L13
2. ```truffle test``` always run compilation before testing

Taking this into account I would propose using ```npm run-script test``` instead of ```npm run-script compile``` to no build contracts twice.

Reply:
I would prefer to have unified way of compiling and testing projects.
In this way it is better to have separate compile and test steps.
I can update the test script to rid of build folder removing.

Reply:
The truffle itself compiles tests and contracts. I will remove the compilation and package steps from the CircleCI script.